R/uploadPackage.R

```diff
@@ -259,11 +331,30 @@ uploadPackageServer <- function(id) {
       }
     )
     
-    # Uploaded packages summary.
+    # Removed/Uploaded packages summary.
     output$upload_summary_text <- renderText({
       req(uploaded_pkgs)
       req(nrow(uploaded_pkgs()) > 0)
-      
+      # modify the message if we are removing packages
+      if(isTruthy(sum(uploaded_pkgs()$status == 'removed') >0)) {
+        loggit::loggit("INFO",
+                       paste("Uploaded file:", input$uploaded_file$name, 
+                             "Removed Packages", sum(uploaded_pkgs()$status == 'removed')),
+                       echo = FALSE)
+        
+        as.character(tagList(
+          br(), br(),
+          hr(),
+          div(id = "upload_summary_div",
+              h5("Summary of Removed package(s)"),
+              br(),
+              p(tags$b("Total Packages: "), nrow(uploaded_pkgs())),
+              p(tags$b("Removed Packages: "), sum(uploaded_pkgs()$status == 'removed')),
```

Comment:
I think it would be nice for "Total Packages" to be the total uploaded packages, and to have another line denoting the remaining number of packages.

Reply:
I replaced "Total Packages" with "Remaining Packages"